starlingcaptureapi/actions.py

```diff
@@ -22,18 +23,102 @@
 class Actions:
     """Actions for processing assets."""
 
-    def archive(self, asset_meta_path: str):
+    def archive(self, organization_id, collection_id, zip_path, asset_exts, aes_key):
         """Archive asset.
 
         Args:
-            asset_meta_fullpath: full local path to the metadata JSON file for this asset
+            organization_id: ID of the organization
+            collection_id: ID of the collection this asset is under
+            zip_path: path to the ZIP that contains the asset and metadata
+            asset_exts: valid file extensions for the asset (no dots)
+            aes_key: 32 byte key for AES-256-CBC encryption
+
+        Returns:
+            TODO
 
         Raises:
             Exception if errors are encountered during processing
         """
-        archive = EncryptedArchive.make_from_meta(asset_meta_path)
-        Iscn.register_archive(archive)
 
+        asset_helper = AssetHelper(organization_id)
+        file_util = FileUtil()
+
+        tmp_dir = asset_helper.get_tmp_collection_dir(collection_id, "archive")
+
+        # Copy ZIP
+        tmp_zip = shutil.copy2(zip_path, tmp_dir)
+
+        # Verify zip contents are valid and expected
+
```

Comment:
Coding style wise I think we don't leave an empty line here elsewhere.

```suggestion
```

Reply:
Fixed in 5c7f64e